mesonbuild/dependencies/dev.py

```diff
@@ -510,7 +510,7 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
                 return
 
             if self.clib_compiler.get_argument_syntax() == 'msvc':
-                libs = ['zlib1' 'zlib']
+                libs = ['zlib1', 'zlib']
```

Comment:
Can we split this out into its own PR, because I think we probably want to backport this...